frontend/interfaces/platform.ts

```diff
@@ -111,3 +111,55 @@ export const isAppleDevice = (platform: string) => {
 
 export const isIPadOrIPhone = (platform: string | HostPlatform) =>
   ["ios", "ipados"].includes(platform);
+
+export const DISK_ENCRYPTION_SUPPORTED_LINUX_PLATFORMS = [
+  "ubuntu", // covers Kubuntu
+  "rhel", // *included here to support Fedora systems. Necessary to cross-check with `os_versions` as well to confrim host is Fedora and not another, non-support rhel-like platform.
+] as const;
+
+export const isDiskEncryptionSupportedLinuxPlatform = (
+  platform: HostPlatform,
+  os_version: string
+) => {
+  const isFedora =
+    platform === "rhel" && os_version.toLowerCase().includes("fedora");
+  return isFedora || platform === "ubuntu";
+};
+
+const DISK_ENCRYPTION_SUPPORTED_PLATFORMS = [
+  "darwin",
+  "windows",
+  "chrome",
```

Comment:
Do we support disk encryption for Chrome OS?

Reply:
The constant name is a bit misleading – it's being used to determine whether to _show_ disk encryption UI, which [we do for Chrome](https://github.com/fleetdm/fleet/blob/9a3931ddefed3324b34c24ad3332481cb251f3ed/frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx#L177)

Reply:
Ah, TIL
![Screenshot 2024-11-20 at 8 45 19 AM](https://github.com/user-attachments/assets/62dfca15-ff29-4029-b852-eb951af6faa4)